src/pages/MoreOptions/NodeSettings/NodeSettingsContainerScreen.tsx

```diff
@@ -76,7 +76,7 @@ const NodeSettingsContainerScreen: React.FC<Props> = ( ) => {
       isConnectionActive: true,
       ipAddress: ipAddress,
       portNumber: portNumber,
-      urlPath: `${ipAddress}:${portNumber}`,
+      urlPath: `http://${ipAddress}:${portNumber}`,
```

Comment:
The user may if may not have https so we need to ensure that it is added by the user as part of the URL and if it’s added it will be part of ipAddress.